### PR TITLE
REDSHOP-6122: Create button clear infamation when reorder.

### PR DESCRIPTION
**Pull request by hoang-tran-redweb**
add button clear user info

### CODE REVIEW

File: component/site/views/checkout/tmpl/default_address.php [hunk 1]
```diff
@@ -37,9 +39,13 @@
...
-                           rel="{handler: 'iframe', size: {x: 720, y: 470}}"><?php echo JText::_(
-                                'COM_REDSHOP_EDIT'
-                            ); ?></a>
+                           rel="{handler: 'iframe', size: {x: 720, y: 470}}"><?php echo JText::_('COM_REDSHOP_EDIT'); ?>
+                        </a>
+	                    <?php if ($auth['users_info_id'] && Redshop::getConfig()->getInt('ENABLE_CLEAR_USER_INFO') == 1): ?>
+                            <button type="button" class="btn btn-primary" name="clear_user_info" onclick="javascript:clearUserInfo();">
+			                    <?php echo JText::_('COM_REDSHOP_CLEAR_USER_INFO'); ?>
+                            </button>
+	                    <?php endif;?>
```
Comment (anhredweb) [inline component/site/views/checkout/tmpl/default_address.php]: @hoang-tran-redweb : `COM_REDSHOP_CLEAR_USER_INFO` not defined language yet.
image

Comment (hoang-tran-redweb) [reply component/site/views/checkout/tmpl/default_address.php]: @anhredweb I fixed. please review again.

**Review by anhredweb — DISMISSED** [review]

**Comment by hoang-tran-redweb** [comment]
@anhredweb I fixed conflict,

**Comment by anhredweb** [comment]
@NguyenTrang091296 please help for checking drone fail case
Thanks
H.A

**Comment by hoang-tran-redweb** [comment]
@anhredweb I fixed scrutinizer. please check help me

**Comment by NguyenTrang091296** [comment]
it pass drone now.Please help me review and merge it. tks cc @anhredweb

**Review by anhredweb — APPROVED** [review]

**Pull request merged**